chainer/functions/array/get_item.py

```diff
@@ -49,10 +53,32 @@ def __init__(self, slices, in_shape, in_dtype):
         self._in_dtype = in_dtype
 
     def forward(self, inputs):
+        gy, = inputs
         xp = cuda.get_array_module(*inputs)
         gx = xp.zeros(self._in_shape, self._in_dtype)
         if xp is numpy:
-            numpy.add.at(gx, self.slices, inputs[0])
+            try:
+                numpy.add.at(gx, self.slices, gy)
+            except IndexError:
+                done = False
+                # In numpy<1.13, 0-dim boolean index is only supported by
+                # arr.__getitem__ for 0-dim arr.
```

Comment:
* Please add that it's not supported in `numpy.add.at`.
* Please add that in such case we're trying to convert the slices to something older NumPy supports.